testng-eclipse-plugin/src/main/org/testng/eclipse/ui/util/ConfigurationHelper.java

```diff
@@ -171,36 +171,18 @@ public static String getJvmArgs(ILaunchConfiguration configuration) throws CoreE
 
     addDebugProperties(jvmArgs, configuration);
 
-    switch (ConfigurationHelper.getProtocol(configuration)) {
-    case STRING:
-      jvmArgs.append(" -Dtestng.eclipse.stringprotocol");

```

Comment:
this property `testng.eclipse.stringprotocol` is invalid since we pass the `-protocol` arg (rather than system properties) to tell the RemoteTestNG the communication protocol.